packages/Optimizer-Swift

```diff
@@ -0,0 +1,35 @@
+layout: "package"
+permalink: "Optimizer-Swift"
```

Comment:
It seems from the package links that this is not Optimizer-Swift package is this still the correct name of the package?